azurerm/internal/services/maintenance/validate/maintenance.go

```diff
@@ -0,0 +1,154 @@
+package validate
+
+import (
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tf/pluginsdk"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tf/validation"
+)
+
+func MaintenanceTimeZone() pluginsdk.SchemaValidateFunc {
```

Comment:
i think we have a similar validation function used elsewhere? searching for "Afghanistan Standard Time" seems we have it twice lol

Reply:
I did see the other list, it was slightly different from the one that PowerShell output though, and the API docs say to use `[System.TimeZoneInfo]::GetSystemTimeZones()`